core/core.go

```diff
@@ -7,10 +7,10 @@
 //   2. Call LoadCorefile() to get the Corefile (it
 //      might have been piped in as part of a restart).
 //      You should pass in your own Corefile loader.
-//   3. Call caddy.Start() to start CoreDNS, caddy.Stop()
-//      to stop it, or caddy.Restart() to restart it.
+//   3. Call core.Start() to start CoreDNS, core.Stop()
+//      to stop it, or core.Restart() to restart it.
 //
-// You should use caddy.Wait() to wait for all CoreDNS servers
```

Comment:
Note: Once I finish: https://github.com/miekg/caddy-coredns all this code will not be needed anyway.